src/workloads/scale/InsertRemove.yml

```diff
@@ -8,7 +8,7 @@ Description: |
 
 Keywords:
 - docs
-- InsertRemove
```

Comment:
This one uses the InsertRemove actor, which is instantiated with "InsertRemove" in the yaml. I think all the actors are named CamelCase as opposed to lowerCamelCase. I don't have a strong opinion, but that's why this one was different.

Reply:
If this keyword indicates the actor name, can we change it to "actorInsertRemove"? Just trying to consolidate the naming convention since all other keywords are using lowerCamelCase.

Reply:
Yeah, that sounds great!